identicon.js

```diff
@@ -163,8 +168,8 @@
             return 'rgba(' + values.join(',') + ')';
         },
 
-        getBase64: function(){
-            var i,
+        getSvgElement: function(){
```

Comment:
You could call this getDump() to match the PNGlib interface. Also, if we are going to output raw XML, we should switch to single quotes here.